ios/RCTContacts/RCTContacts.mm

```diff
@@ -66,7 +66,7 @@ - (NSDictionary *)constantsToExport
     } else if (authStatus == CNAuthorizationStatusAuthorized){
         resolve(@"authorized");
     } else if(@available(iOS 18, *)) {
-        if (authStatus == CNAuthorizationStatusLimited) {
+        if (authStatus == PHAuthorizationStatusLimited) {
```

Comment:
Don't you need to import `PHAuthorizationStatisLimited` at the top?

Reply:
Oh think i did something wrong ☠️
Well i am not ios developer these are xcode suggestions so its worked in my current project with new arch so thats why i added this code.

Reply:
I'm going to close this PR since you are not an ios developer. If you are not sure then it may just make things more confusing.